<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FplTelegramWidget</class>
 <widget class="QWidget" name="FplTelegramWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Numder of aircrafts:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_20">
     <property name="text">
      <string>7.</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QSpinBox" name="speed">
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLineEdit" name="aircraftType">
     <property name="maxLength">
      <number>7</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_22">
     <property name="text">
      <string>9.</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_24">
     <property name="text">
      <string>13.</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Departure aerodrome:</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_27">
     <property name="text">
      <string>18.</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="numberOfAircrafts">
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="telegramType">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>FPL</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Destination aerodrome:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_21">
     <property name="text">
      <string>8.</string>
     </property>
    </widget>
   </item>
   <item row="12" column="4">
    <widget class="QLineEdit" name="eet">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="inputMask">
      <string>99:99;_</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Flight rules:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QComboBox" name="flightType"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_25">
     <property name="text">
      <string>15.</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Other:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QComboBox" name="speedType"/>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Speed:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>WTC:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="8">
    <widget class="QTextEdit" name="route">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="15" column="1" colspan="8">
    <widget class="QTextEdit" name="other">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="10" column="8">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="selectRoute">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="4">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>Flight level:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Telegram  type:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="flightRules"/>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Flight type:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QComboBox" name="wtc"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_19">
     <property name="text">
      <string>3.</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Departure time:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="3">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>EET:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_26">
     <property name="text">
      <string>16.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="acid">
     <property name="maxLength">
      <number>7</number>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Equipment:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_23">
     <property name="text">
      <string>10.</string>
     </property>
    </widget>
   </item>
   <item row="12" column="5">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>Alt. aerodrome 1:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="6">
    <widget class="QSpinBox" name="flightLevel">
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Route:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>ACID:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="7">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>Alt. aerodrome 2:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QTimeEdit" name="departureTime">
     <property name="displayFormat">
      <string>HH:mm</string>
     </property>
    </widget>
   </item>
   <item row="7" column="5">
    <widget class="QComboBox" name="flightLevelType"/>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="adep">
     <property name="maxLength">
      <number>7</number>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="equipment">
     <property name="maxLength">
      <number>7</number>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QLineEdit" name="ades">
     <property name="maxLength">
      <number>7</number>
     </property>
    </widget>
   </item>
   <item row="12" column="6">
    <widget class="QLineEdit" name="alt1">
     <property name="maxLength">
      <number>7</number>
     </property>
    </widget>
   </item>
   <item row="12" column="8">
    <widget class="QLineEdit" name="alt2">
     <property name="maxLength">
      <number>7</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>acid</tabstop>
  <tabstop>flightRules</tabstop>
  <tabstop>flightType</tabstop>
  <tabstop>numberOfAircrafts</tabstop>
  <tabstop>wtc</tabstop>
  <tabstop>departureTime</tabstop>
  <tabstop>speedType</tabstop>
  <tabstop>speed</tabstop>
  <tabstop>flightLevelType</tabstop>
  <tabstop>flightLevel</tabstop>
  <tabstop>selectRoute</tabstop>
  <tabstop>route</tabstop>
  <tabstop>eet</tabstop>
  <tabstop>other</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
